### PR TITLE
OSDOCS-5694: Documented a golang known issue for vSphere

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-5694](https://issues.redhat.com/browse/OSDOCS-5694)

Version(s):
4.13

Link to docs preview:
[Preview](https://58691--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-known-issues)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
A few question/suggestion based on the conversation in JIRA

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -1750,6 +1750,7 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 Workaround: Add the mandatory `spec.git_provider.user` field for Bitbucket. In addition, select either *Git access token* or *Git access token secret* to continue adding a Git Repository. (link:https://issues.redhat.com/browse/OCPBUGS-7036[*OCPBUGS-7036*])
 
+* Currently, a certificate compliance issue, specifically outputted as `x509: certificate is not standards compliant`, exists when you run the `openshift-installer` on a macOS for the purposes of installing an {product-title} cluster on VMware vSphere. This issue relates to a known issue with the `golang` compiler in that the compiler does not recognize newly supported macOS certificate standards. No workaround exists for this issue. (link:https://issues.redhat.com/browse/OSDOCS-5694[*OSDOCS-5694*])
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-13-release-notes.adoc:1753]: Hi @JoeAldinger . Not sure what you mean by a specification. Do you have an example?

Regarding, dropping the "upgrade" statement, can you access the following Slack thread?

[# forum-installer](https://redhat-internal.slack.com/archives/C68TNFWA2/p1681458419000539?thread_ts=1681393056.508109&cid=C68TNFWA2)

**Pull request merged**